package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@doc-tools/transform": "^3.11.0",
     "@gravity-ui/components": "2.1.0",
-    "@gravity-ui/date-components": "^1.3.0",
+    "@gravity-ui/date-components": "^1.4.0",
```

Comment:
@Arucard89 here is a problem with displaying the "Size" block:

<img width="944" alt="image" src="https://github.com/gravity-ui/landing/assets/29424746/8c7d0086-a8eb-4fa3-9e07-c9ec87239c00">

Can we place them in separate blocks?

Reply:
@imsitnikov , hi. I've checked other components and found that the problem is not the only one. Other components have the same problem with positioning of items in row. For Example in Button component:
![image](https://github.com/gravity-ui/landing/assets/13587201/497e7cce-ba9b-4f6a-be2f-4762b80cb9eb)
So I want to suggest another decision. What do you think about changing css to wrap element automatically?
https://github.com/gravity-ui/landing/pull/167/commits/f334d5a7139b3df7c4d67e4610ba53c6ee30cb71

Reply:
This is a good idea, but can we also add `align-items: center;` and `justify-content: center;`?

Reply:
Added: https://github.com/gravity-ui/landing/pull/167/commits/8f71cf414930bfe77d25146c8c7e5d162bf1b46b